encryptor/searchable_query_filter_test.go

```diff
@@ -0,0 +1,74 @@
+package encryptor
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/cossacklabs/acra/encryptor/config"
+	"github.com/cossacklabs/acra/sqlparser"
+)
+
+func TestGetTableSchemaOfColumnMatchConfiTable(t *testing.T) {
+	tableName := "some_table"
```

Comment:
can we use name with upper-cased characters here and below? to be sure that we handle it correctly. Because previously we handled it in lowercase by default